LiveSplit.AutoSplitters.xml

```diff
@@ -19072,7 +19072,8 @@
             <Game>Gothic 2 Gold</Game>
         </Games>
         <URLs>
-            <URL>https://raw.githubusercontent.com/thekovic/ASL-Autosplitters/main/Gothic.asl</URL>
+            <URL>https://raw.githubusercontent.com/thekovic/ASL-Autosplitters/main/Gothic.asl</URL>            
+            <URL>https://raw.githubusercontent.com/Nuget94/ASL-Autosplitters/main/gothic2.asl</URL>
```

Comment:
```suggestion        
            <URL>https://raw.githubusercontent.com/Nuget94/ASL-Autosplitters/main/gothic2.asl</URL>
```
There can only be one script linked.

Reply:
Should be all right. my script also includes the functionality of the old script.

Reply:
My understanding is that LiveSplit will download all the files, but it will only load the first URL in the list. Why do you need to download both scripts?

Reply:
I think OP is being overly polite to me. The old link should be removed for correct functionality as the new implementation replaces it entirely.